packages/webdriverio/src/middlewares.js

```diff
@@ -36,6 +36,12 @@ export const elementErrorHandler = (fn) => (commandName, commandFn) => {
 
                     return await fn(commandName, commandFn).apply(this, args)
                 }
+                if (typeof error === CustomRequestError) {
+                    throw new Error(`
```

Comment:
by re-throwing the error I think we loose the stacktrace here. Can't we just change the error message?